apps/web-giddh/src/app/gst/gstR1/gstR1.component.ts

```diff
@@ -9,10 +9,39 @@ declare var $: any;
     styleUrls: ['gstR1.component.scss'],
 })
 export class FileGstR1Component implements OnInit {
+    /* This will hold the value out/in to open/close setting sidebar popup */
+    public asideInventorySidebarMenuState: string = 'in';
+    /* Aside pane state*/
+    public asideMenuState: string = 'out';
+    /* this will check mobile screen size */
+    public isMobileScreen: boolean = false;
+
     constructor() {
         //
     }
 
+    /* Aside pane toggle fixed class */
+    public toggleBodyClass(): void {
+        if (this.asideMenuState === 'in') {
+            document.querySelector('body').classList.add('fixed');
+        } else {
+            document.querySelector('body').classList.remove('fixed');
+        }
+    }
+    /**
+      * This will toggle the settings popup
+      *
+      * @param {*} [event]
+      * @memberof SettingsComponent
```

Comment:
Update @memberof and name of method as it toggles Gst side pane and not settings pane
```suggestion
      * @memberof FileGstR1Component
```

Reply:
Done